templates/pg_hba.conf.j2

```diff
@@ -79,7 +79,7 @@ local   all             all                                     peer
 
 # IPv4 local connections:
 {% if postgresql_use_ssl %}
-hostssl    {{ postgresql_database.database_name }}            {{ postgresql_database.user_name }}            {{ postgresql_connection.client_ip }}    255.255.255.255    md5    clientcert=1
+hostssl    {{ postgresql_database.database_name }}            {{ postgresql_database.user_name }}            {{ postgresql_connection.client_ip }}    {{ postgresql_connection.subnet_mask|default(255.255.255.255) }}    md5    clientcert=1
```

Comment:
Based on the [CI error](https://github.com/UCL-MIRSG/ansible-role-postgresql/actions/runs/4297480888/jobs/7500862306#step:5:377), I think the default subnet mask might need to be a string

Reply:
Changed as suggested.

Reply:
Got the tests to pass by explicitly setting `listen_addresses` to `'*'` for Molecule (I guess `0.0.0.0` would also work).

BTW, these changes were to enable deploying on Harvester rather than being fixes.